### PR TITLE
feat: add Gitea module

### PR DESCRIPTION
**Pull request by alex-karpenko**
This is a module for `Gitea` self-hostded git server.
It's quite usefull to test various git clients with http/https/ssh access to public and private git repos.

Gitea documentation: https://docs.gitea.com/
Gitea repo: https://github.com/go-gitea/gitea

### CODE REVIEW

**Comment by alex-karpenko** [comment]
Looks like all checks completed successfully, but workflow failed on post-action step due to lack of runner's storage.

File: Cargo.toml [hunk 1]
```diff
@@ -58,7 +59,12 @@ pulsar = []
 
 [dependencies]
 # TODO: update parse-display after MSRV>=1.80.0 bump of `testcontainer-rs` and `testcontainers-modules`
-parse-display = { version = "0.9.1", optional = true, default-features = false, features = [] }
+parse-display = { version = "0.9.1", optional = true, default-features = false, features = [
```
[the remark this answers is not in the document]
Comment (alex-karpenko) [reply Cargo.toml]: Yeah, i noticed and fixed. Thank you.

**Review by mervyn-mccreight — APPROVED** [review]

**Review by DDtKey — APPROVED** [review]

**Comment by DDtKey** [comment]
Thank you for the contribution! 🙏

**Comment by mervyn-mccreight** [comment]
@alex-karpenko 

This feature has been released right now and is available with 0.11.4
Sorry for the delay.

**Pull request merged**